src/Symfony/Component/Form/Extension/Core/DataTransformer/UlidToStringTransformer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\Uid\Ulid;
+
+/**
+ * Transforms between a ULID string and a Ulid object.
+ *
+ * @author Pavel Dyakonov <wapinet@mail.ru>
+ */
+class UlidToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * Transforms a Ulid object into a string.
+     *
+     * @param Ulid $value A Ulid object
+     *
+     * @return string A value as produced by Uid component
+     *
+     * @throws TransformationFailedException If the given value is not a Ulid object
+     */
+    public function transform($value)
+    {
+        if (null === $value) {
+            return '';
```

Comment:
I was about to improve but rereading the code and the tests I think we should rather return `null` instead of the empty string here (same in the `UuidToStringTransformer` class).

Reply:
fixed